assets/src/edit-story/elements/text/test/output.js

```diff
@@ -139,6 +139,63 @@ describe('TextOutput', () => {
     expect(output.innerHTML).toBe('<strong>Content</strong>');
   });
 
+  it('should wrap font-family into quotes', () => {
+    const element = {
+      id: '123',
+      content: 'Content',
+      type: 'text',
+      x: 10,
+      y: 10,
+      width: 50,
+      height: 50,
+      fontFamily: 'Baloo Bhaina 2',
+      rotationAngle: 0,
+      padding: {
+        vertical: 0,
+        horizontal: 0,
+      },
+      fontSize: 16,
+      color: { type: 'solid', color: { r: 255, g: 255, b: 255 } },
+    };
+
+    const output = renderViaString(
+      <TextOutput
+        element={element}
+        box={{ width: 50, height: 50, x: 10, y: 10, rotationAngle: 0 }}
+      />
+    );
+    expect(output.style.fontFamily).toBe('"Baloo Bhaina 2"');
+  });
+
+  it('should wrap font fallback into quotes', () => {
+    const element = {
+      id: '123',
+      content: 'Content',
+      type: 'text',
+      x: 10,
+      y: 10,
+      width: 50,
+      height: 50,
+      fontFamily: 'Baloo Bhaina 2',
+      fontFallback: ['cursive'],
+      rotationAngle: 0,
+      padding: {
+        vertical: 0,
+        horizontal: 0,
+      },
+      fontSize: 16,
+      color: { type: 'solid', color: { r: 255, g: 255, b: 255 } },
+    };
+
+    const output = renderViaString(
+      <TextOutput
+        element={element}
+        box={{ width: 50, height: 50, x: 10, y: 10, rotationAngle: 0 }}
+      />
+    );
+    expect(output.style.fontFamily).toBe('"Baloo Bhaina 2","cursive"');
```

Comment:
Oh, this is actually a bug!

`font-family: "cursive"` refers to a font named "cursive", not to the generic family keyword. To reference generic family keywords, font names **have to be identifiers**, not strings, and thus must never be quoted (as per [@swissspidy's link](https://github.com/google/web-stories-wp/pull/1098#issuecomment-610831875) to [work by Bynens](https://mathiasbynens.be/notes/unquoted-font-family)).

You need this whitelist in your util function and make sure to never quote these font names:

```js
const genericFamilyKeywords = 'serif,sans-serif,cursive,fantasy,monospace',split(,); // inherit makes no sense here
```

Reply:
Oh!!! And I just read it 🤦‍♀